docs/publishing-artifacts.md

```diff
@@ -48,4 +39,24 @@ But please consider using automatic rules instead of explicitly specifying `publ
  ```json
  "release": "build"
  ```
- and if you run `yarn release`, a release will be drafted (if doesn't already exist) and artifacts published.
\ No newline at end of file
+ and if you run `yarn release`, a release will be drafted (if doesn't already exist) and artifacts published.
+ 
+## Recommended GitHub-releases workflow
+
+  **NOTICE**: _This is the recommended workflow._
+
+ 1. [Draft a new release](https://help.github.com/articles/creating-releases/). Set the "Tag version" to the value of `version` in your application `package.json`, and prefix it with `v`. "Release title" can be anything you want.
+   - For example, if your application `package.json` version is `1.0`, your draft's "Tag version" would be `v1.0`.
+ 2. Push some commits. Every CI build will update the artifacts attached to this draft.
+ 3. Once you are done, publish the release. GitHub will tag the latest commit for you.
+
+ The benefit of this workflow is that it allows you to always have the latest artifacts, and the release can be published once it is ready.
+ 
+## S3 workflow
+
+If you are using S3 instead of GitHub releases to publish artifacts the above workflow won't work. This workflow is modelled on how releases are handled in maven.
+
+1. Setup your CI to publish on each commit. E.g. `"dist": "electron-builder --publish always"` in your package.json.
+2. Set your version in package.json to `1.9.0-snapshot` (or `1.9.0-master` or whatever you want your development channel to be named). This will publish a file named `master.yml` and a build named `something-master.exe` (and corresponding for mac) to S3.
```

Comment:
if `1.9.0-snapshot`, so, later in the doc we should use `snapshot`, and not `master` :)

Reply:
Whoopsie, fixed :)